src/resources/2014/ConspiracyIsland.ts

```diff
@@ -0,0 +1,296 @@
+import {
+  canInteract,
+  Item,
+  Location,
+  Monster,
+  runChoice,
+  visitUrl,
+} from "kolmafia";
+import { get } from "../../property";
+import { $item, $location, $monster } from "../../template-string";
+
+const radio = "place.php?whichplace=airport_spooky&action=airport2_radio";
+const omegapanel =
+  "place.php?whichplace=airport_spooky_bunker&action=si_controlpanel";
+
+export function have(): boolean {
+  return get("spookyAirportAlways");
+}
+
+class QuestData {
```

Comment:
there's only one method on this class; it may be worth making this an object type instead.